compoments/report-repair/repair-picture.js

```diff
@@ -0,0 +1,121 @@
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+import ErrorSummary from '../errorSummary';
+import Button from '../button';
+import {serviceName} from '../../helpers/constants';
+import imageToBase64 from 'image-to-base64/browser';
+
+const RepairPicture = ({ handleChange, values }) => {
```

Comment:
This name suggests it's a component that represents a picture of the repair, whereas the intent of this component is to upload a photo/image. Please update the name to represent the intent of the component, i.e. `RepairImageUpload`